dotnet-aspnet/Dockerfile.24.10

```
@@ -21,7 +21,9 @@ RUN mkdir /rootfs \
         base-files_release-info \
         base-files_chisel \
         ca-certificates_data \
-        aspnetcore-runtime-8.0_libs
+        aspnetcore-runtime-9.0_libs
+RUN mv /rootfs/usr/lib/dotnet/dotnet9 rootfs/usr/lib/dotnet/dotnet \
```

```suggestion
RUN mv /rootfs/usr/lib/dotnet/dotnet9 /rootfs/usr/lib/dotnet/dotnet \
```

nitpick: to keep the paths consistent.